utils/manifest-utils/src/rules.rs

```diff
@@ -0,0 +1,146 @@
+use std::{
+    fs::OpenOptions,
+    io::BufReader,
+    path::{Path, PathBuf},
+    sync::{Arc, RwLock},
+};
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use structopt::StructOpt;
+
+#[derive(Clone, Debug, Default)]
+pub struct RuleStore {
+    pub path: PathBuf,
+    config: Arc<RwLock<RulesConfig>>,
+}
+
+impl RuleStore {
+    pub fn load_or_create(rules_file: &Path) -> Result<Self> {
+        if rules_file.exists() {
+            log::debug!("Loading rule from: {:?}", rules_file);
```

Comment:
You can use `.display()` on `Path` and `PathBuf`
I would be rather against using debug prints in code if not necessary, because they can be ugly